rfcs/text/0001-rfc-process.md

```diff
@@ -70,103 +84,111 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
+    - Add a two week _last call_ for the completion of the Science review to the top-level summary comment in the RFC pull request.
 
     **EXAMPLE** 
 
     **April 5**: Last call for science review
 
-   **NOTE**: *The Science review occurs in parallel to the Community review.* 
+**NOTE**: *The Science review occurs in parallel to the Community review.* 
 
 ### Reviewing RFCs
 
 #### Approvers:
-- May assign specific reviewers in the RFC pull request
+- MAY assign specific reviewers in the RFC pull request.
 
 #### Reviewers:
-- Add feedback to the RFC pull request comment thread
+- Add feedback to the RFC pull request comment thread.
 
 #### Science Reviewer:
 - When the feedback from the Science review is addressed, replace the "science-review-required" label with "science-review-completed".
 
 #### Author: 
-- Revise the RFC pull request in response to feedback and push new commits
+- Revise the RFC pull request in response to feedback and push new commits.
 
 ### Shepherding RFCs
 [Shepherding RFCs]: #shepherding-rfcs
 #### Shepherd:
 - Monitor the community review and ensure that issues are addressed by the **Author(s)** in a reasonable time period.
 
-- When all issues are addressed and any required Science reviews are complete:
-  - Summarize the review discussion for the **Approvers** in the top-level summary comment in the RFC pull request
-  - Add a minimum one week _last call_ for the completion of the Oversight review to the top-level summary comment in the RFC pull request.
-
-    **EXAMPLE** 
-
-    ~~**April 1**: Last call for community review~~
-
-    **April 22**: Last call for oversight review
-
-  - Replace "rfc-community-review" with "rfc-oversight-review"
+- For Software RFCs, add the RFC as an agenda item to the next *DCP Architecture* meeting.
 
-  **NOTE**: *Oversight review is limited to **Approvers**. Further community reviews during this period may be disregarded by the **Author(s)**.*
+- For Governance RFCs, add the RFC as an agenda item to the next *DCP PM* meeting.
 
-- For software RFCs, request an Oversight review of the RFC on the HumanCellAtlas **#tech-architecture** slack channel. Include a link to the RFC pull request and the last call deadline
-
-  Add the RFC as an agenda item to the next *DCP Architecture* meeting
-
-- For governance RFCs, request an Oversight review of the RFC on the HumanCellAtlas **#dcp-project-mgmt** slack channel. Include a link to the RFC pull request and the last call deadline:
-
-    **EXAMPLE** 
-  
-     ***@channel**: Call for oversight review of the RFC process - https://github.com/HumanCellAtlas/dcp-community/pull/27 - Last call is April 22.*
+- When all issues are addressed and any required Science reviews are complete:
 
-  Add the RFC as an agenda item to the next *DCP PM* meeting
+  - Summarize the review discussion for the **Approvers** in the top-level summary comment in the RFC pull request.
```

Comment:
We could now include an example from the set of approved RFCs to demonstrate what this intends.

Reply:
Done! I added a link instead of explicitly writing it out if that sounds ok?